database/migrations/2024_09_27_100548_add_pause_field_to_tasks_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('tasks', function (Blueprint $table) {
+            $table->boolean('pause')->default(false);
```

Comment:
this does not match the name of the field you used in the RunAllReadyTasks
```suggestion
            $table->boolean('paused')->default(false);
```